src/core/solutions/brown_function.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/* 
+ * File:   brown_function.cpp
+ * Author: Alejandro Ruiz Madruga <amadruga at estudiantes.uci.cu>
+ *
+ * Created on 28 de septiembre de 2019, 21:56
+ */

```

Comment:
Consider that cpp files do not require this heading. This is for .h files only

Reply:
It could be, let me update the repository with the Csendes function implementation and then create the directory to clonate again the repository

Reply:
@alejandrom247 remember that to update changes you don't need to clone the repo again, just modify the code, make a regular commit and it will be updated here